src/test/java/pages/RoomPage.java

```diff
@@ -7,7 +7,7 @@ public class RoomPage {
     public static By calendarTable = By.xpath("//div[@class='calendar-table']");
     public static By invalidDate = By.xpath("(//div[@class='calendar-table']//td[contains(@class,'disabled')])[1]");
```

Comment:
call this disabled date, invalid is for a particular test case

Reply:
Done